content/tutorials/single-page-tutorials/create-nft-with-wallet-lib.md

```diff
@@ -0,0 +1,197 @@
+# Create a NFT with wallet library and IPFS
+
+In this tutorial you will upload your desired image to IPFS and create a NFT from the unique IPFS URL on the Shimmer testnet. This tutorial will build on the tutorial "Setup testnet address and receive funds from faucet" ([IOTA Wiki PR under review](https://github.com/iota-community/iota-wiki/pull/711)). If you haven't completed it, please do so and afterwards come back here. You will use the same folder and development environment as in the mentioned prerequired tutorial, which will allow you to reuse the Stronghold account with funds on it.
+
+## Prerequisites
+
+- Installed [Node.js](https://nodejs.org/en/)
+- Finished [Setup testnet address and receive funds from faucet](https://wiki.iota.org/wallet-setup) tutorial

```

Comment:
```suggestion
- [Node.js](https://nodejs.org/en/).
- Finish the [Set up testnet address and receive funds from faucet](https://wiki.iota.org/wallet-setup) tutorial.
```
just in case, do we require a specific/minimum node version?

Reply:
Not sure. But I don't think so 🤔 . Let me ask